doc/source/working_with_kiwi.rst

```diff
@@ -84,3 +85,149 @@ to find in the *description directory*:
           <archive name="custom-archive.tgz"/>
       </packages>
 
+
+.. _working-with-kiwi-image-building-process:
+
+Image Building Process
+----------------------
+
+KIWI creates images in a two step process. The first step, the *prepare*
+operation, generates a so-called *unpacked image* tree (directory) using
+the information provided in the :file:`config.xml` configuration file
+(see :ref:`xml-description`)
+
+The second step, the *create* operation, creates the *packed image* or
+*image* in the specified format based on the unpacked image and the
+information provided in the :file:`config.xml` configuration file.
+
+.. figure:: .images/intro.png
+    :align: center
+    :alt: Image Creation Architecture
+
+    Image Creation Architecture
+
+
+.. _prepare-step:
+
+The Prepare Step
+^^^^^^^^^^^^^^^^
+
+The first step of the image creation is the *prepare* step, during which
+KIWI creates an *unpacked image*, also called the "root tree". This
+directory will be the installation target for software packages to be
+installed during the image creation process.
+
+For the package installation, KIWI relies on the package manager specified
+in the ``packagemanager`` element in :file:`config.xml` (see
+:ref:`xml-description-preferences-common-elements`). KIWI supports the
+following package managers: ``dnf``, ``zypper`` (default) and ``apt-get``.
+
+The prepare step consists of the following substeps:
+
+#. **Create Target Root Directory.**
+
+   KIWI will exit with an error if the target root tree already exists to
+   avoid accidental deletion of an existing unpacked image.
+
+#. **Install Packages.**
+
+   KIWI first configures the package manager to use the repositories
+   specified in the configuration file and/or via the command line. After
```

Comment:
I'm not completely sure if I understood it correctly. So you basically say, you can use either one or both methods, right? If this is the case, I would suggest:

```suggestion
   specified in the configuration file, via the command line, or both. After
```

Reply:
I actually don't know that. @schaefi?

Reply:
On the command line you can use the options ```--add-repo``` which adds a repo to the existing ones. You can also use ```--set-repo``` which overwrites the first repo. A combination of the options is possible but in most cases not useful. Usually people use --set-repo if the XML description just has one repo and they want to overwrite that one.

Reply:
Actually the proposed change of the documentation looks good to me. For details on the command line the man page explains for the system build command explains it pretty precisely. So I would go with the suggestion from Tom